bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/Alloc.java

```diff
@@ -174,7 +175,7 @@ private synchronized void initCircumstances() {
 		if (circV == null) {
 
 			// Calculate visit circumstances, which includes setup time.
-			WorldCoords coords = new WorldCoords(getObs().getRa(), getObs().getDec());
+			WorldCoords coords = getObs().getCoords(getMiddlePoint());
```

Comment:
Here again we get coords at the middle point and then use them later on (line 188) at a point where we have a better idea of the precise time for which we wanted the coordinates (and worse, it is changing over a loop).  We should use the most accurate time possible for getting the coordinates in all cases.